bc_obps/reporting/migrations/00010_report_data_tables.py

```diff
@@ -0,0 +1,692 @@
+                ('archived_at', models.DateTimeField(blank=True, null=True)),
+                (
+                    'json_data',
+                    models.JSONField(db_comment='A flat JSON object representing the data collected for this model'),
+                ),
+                (
+                    'activity',
+                    models.ForeignKey(
+                        db_comment='The reporting activity this data applies to',
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_records',
+                        to='registration.reportingactivity',
+                    ),
+                ),
+                (
+                    'activity_base_schema',
+                    models.ForeignKey(
+                        db_comment='The activity base schema used to render the form that collected this data',
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_records',
+                        to='reporting.activityjsonschema',
+                    ),
+                ),
+                (
+                    'archived_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_archived',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'created_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_created',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'report_version',
+                    models.ForeignKey(
+                        db_comment='The report version this data belongs to',
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name='%(class)s_records',
+                        to='reporting.reportversion',
+                    ),
+                ),
+                (
+                    'updated_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_updated',
+                        to='registration.user',
+                    ),
+                ),
+            ],
+            options={
+                'db_table': 'erc"."report_activity',
+                'db_table_comment': 'A table to store the reported activity-specific data, in a JSON format',
+            },
+        ),
+        migrations.CreateModel(
```

Comment:
I don't see the fkeys to the unit / source type data in the migration here (though they are defined in the model?)

Reply:
I think it's because of how django creates the models, the "report source type" model doesn't exist when this code is run yet. Once the model is added later on, we can add the fkeys (see Line 596 and 681). 

TLDR; the fields are there, just further down the file